geopackage_validator/validations/geometry_empty_check.py

```diff
@@ -0,0 +1,45 @@
+from typing import Iterable, Tuple
+from geopackage_validator.validations import validator
+from geopackage_validator import utils
+
+SQL_EMPTY_TEMPLATE = """SELECT count(row_id) AS count, row_id
```

Comment:
is deze count wel eens hoger dan 1? Aangezien je zowel count als row_id selecteert en ik neem aan dat elk row_id uniek is?

Reply:
Ja, ik weet ook niet helemaal waarom dit werkt, maar hoe dit uitpakt is dat je de count terugkrijgt en de eerste waarde van row_id. Kun je ook zien in de unittest, de count die terugkomt is 45, met row_id 129.